src/pybind/mgr/orchestrator/module.py

```diff
@@ -1146,6 +1148,52 @@ def _apply_iscsi(self,
 
         return self._apply_misc([spec], dry_run, format, no_overwrite)
 
+    @_cli_write_command('orch apply snmp-gateway')
+    def _apply_snmp_gateway(self,
+                            snmp_version: SNMPVersion,
+                            destination: str,
+                            port: int = 9464,
```

Comment:
```suggestion
                            port: Optional[int] = None,
```
Let#s avoid setting the default value here.